en-US/firefox-ios.xliff

```diff
@@ -30,21 +30,6 @@
         <target>This lets you save photos.</target>
         <note>Privacy - Photo Library Additions Usage Description</note>
       </trans-unit>
-      <trans-unit id="ShortcutItemTitleNewPrivateTab" xml:space="preserve">
-        <source>New Private Tab</source>
```

Comment:
Confirmed with product that v105 strings were imported for good, so we can remove these, and use automation without manual interventions.